### DISCUSSION TITLE
tts-az-1 还是503

### DISCUSSION THREAD

**Discussion by daifai** [BurnHair/docs#140] [技术支持]
POST https://burn.hair/v1/audio/speech
Authorization: Bearer sk-xxxxxxxxxxxx
Content-Type: application/json

{
    "model": "tts-az-1",
    "input": "Hello world",
    "voice": "zh-CN-XiaoxiaoMultilingualNeural"
}

HTTP/1.1 503 Service Unavailable
{
  "error": {
    "message": "当前分组 default 下对于模型 tts-az-1 无可用渠道 (request id: dddddddd)",
    "type": "one_api_error"
  }
}
————————
确定所有配置是正确，sk也是有效的

**Comment by BennyThink** [comment]
嗯 这个非 VIP 分组暂时不能用，还在调整中。

**Reply by daifai** [reply]
意思是只要充值，就能进VIP了呗

**Reply by BennyThink** [reply]
对。 最近在忙一直没空整理

**Reply by daifai** [reply]
我信了你的邪，充值了后，还是报错

{"error":{"message":"无效的令牌 (request id: 20240704080113900963089KQtO3b92)","type":"one_api_error"}}

应该是你的one api的key失效了。快点修复一下。感谢

**Reply by BennyThink** [reply]
我的错，后段代码改错了现在是500，等我修一修

**Reply by daifai** [reply]
对，现在是500了，哈哈，蟹蟹啦

**Reply by BennyThink** [reply]
嗯是个 bug 我这几天修一下

**Reply by BennyThink** [reply]
修好了哦
